apps/docs/content/guides/platform/your-monthly-invoice.mdx

```diff
@@ -0,0 +1,77 @@
+
+### Usage based fees
+Fees vary depending on usage and are paid in arrears. The more usage you have, the higher the fee. Examples are fees for monthly active users and storage size.
+
+### Discounted line items
+Paid plans come with a free quota for certain line items. You only pay for usage that goes beyond the quota. The quota for Storage for example is 100 GB. If you use 105 GB, you pay for 5 GB. If you use 95 GB, you pay nothing. This quota is declared as a discount on your invoice.
+
+#### Compute Credits
+Paid plans come with $10 in Compute Credits per month. This suffices for a single project using a Nano or Micro compute instance. Every additional project adds compute fees to your monthly invoice though.
+
+### Example invoice
+The following invoice was issued on Jan 6, 2025 with the previous billing cycle from Dec 6, 2024 - Jan 5, 2025, and the current billing cycle from Jan 6 - Feb 5, 2025.
+
+<Image
+  alt="Example Invoice"
+  src={{
+    light: '/docs/img/guides/platform/example-invoice.png',
+    dark: '/docs/img/guides/platform/example-invoice.png',
+  }}
+  zoomable
+/>
+
+1) The final amount due
+2) Fixed subscription fee for the current billing cycle
+3) Usage based fee for Compute for the previous billing cycle. There were two projects (wsmmedyqtlrvbcesxdew, wwxdpovgtfcmcnxwsaad) running 744 hours (24 hours * 31 days). These projects incurred $10 in Compute fees each. With $10 in Compute Credits deducted, the final Compute fees are $10.
+4) Usage based fee for Custom Domain for the previous billing cycle. There is no free usage quota for Custom Domain. You get charged for the 744 hours (24 hours * 31 days) a Custom Domain was active. The final Custom Domain fees are $10.19.
+5) Usage based fee for Egress for the previous billing cycle. There is a free usage quota of 250 GB for Egress. You get charged for usage beyond 250 GB only, meaning for 2,119.47 GB. The final Egress fees are $190.75.
+6) Usage based fee for Monthly Active Users for the previous billing cycle. There is a free usage quota of 100,000 users. With 141 users there is no charge for this line item.
+
+### Why is your invoice more than $25?
+The amount due of your invoice being higher than the $25 subscription fee for the Pro Plan can have several reasons.
+
+- **Running several projects:** You had more than one project running in the previous billing cycle. Supabase provides a dedicated server and database for every project. That means that every project you launch incurs compute costs. While the $10 Compute Credits cover a single project using a Nano or Micro compute instance, every additional project adds at least $10 compute costs to your invoice.
+- **Usage beyond quota:** You exceeded the included usage quota for one or more line items in the previous billing cycle while having the Spend Cap disabled.
```

Comment:
The first time we're mentioning spend cap, we might wanna link to the spend cap docs?

Reply:
Yeah. I'm working on the page for the Spend Cap / Cost Management right now. I'll add a respective link then.